Changelog.md

```diff
@@ -2,6 +2,79 @@
 
 ### Ignition Gazebo 6.X.X (202X-XX-XX)
 
+### Ignition Gazebo 6.5.0 (202X-XX-XX)
+1. New trajectory follower system
+    * [Pull request #1332](https://github.com/ignitionrobotics/ign-gazebo/pull/1332)
+
+1. Extend ShaderParam system to support textures
+    * [Pull request #1310](https://github.com/ignitionrobotics/ign-gazebo/pull/1310)
+
+1. Adds a `Link::SetLinearVelocity()` method
+    * [Pull request #1323](https://github.com/ignitionrobotics/ign-gazebo/pull/1323)
+
+1. Fix weird indentation in `Link.hh`
+    * [Pull request #1324](https://github.com/ignitionrobotics/ign-gazebo/pull/1324)
+
+1. fix buoyancy test by increasing tol
+    * [Pull request #fix buoyancy test by increasing tol](https://github.com/ignitionrobotics/ign-gazebo/pull/fix buoyancy test by increasing tol)
```

Comment:
this is from a commit done during a forward port in https://github.com/ignitionrobotics/ign-gazebo/pull/1322. We can remove this.

Reply:
[fcb02cb](https://github.com/ignitionrobotics/ign-gazebo/pull/1338/commits/fcb02cb56f8e50e743c70f03fcf43a52008de0d9)